addons/website/static/src/client_actions/website_preview/website_preview.js

```diff
@@ -364,6 +364,11 @@ export class WebsitePreview extends Component {
             }
         });
         this.iframe.el.contentDocument.addEventListener('keydown', ev => {
+            if (ev && ev.key === 'k' && (ev.ctrlKey || ev.metaKey) && !this.websiteContext.edition) {
```

Comment:
It feels weird to have to re-think ourselves about they mac case, investigating a bit, can't we use the `command_service.js` tools and just check for `control+k` through their utils? (probably `getActiveHotkey()`)

Reply:
Good idea, I adapted my commit to use that util 👍🏻